aqua/LyX/Portfile

```diff
@@ -61,30 +61,35 @@ if {![variant_isset qt4]} {
 }
 
 # Make the default match boost's default.
```

Comment:
see the comment above, `boost` still uses python38 as default so likely the default should not be changed here yet (although adding a python39 variant would be fine).

Reply:
i amended the pr leaving python38 as default (hopefully the right way this time ;-))

Reply:
ah, I didn't notice that and that commit should have closed the associated [Trac ticket](https://trac.macports.org/ticket/62871)... My reading of Ryan's comment there was that when doing that all dependent ports should be updated to use Python 3.9 as well. I assumed that this would be done in the same PR or commit instead of doing this after the fact. But since that didn't happen, changing this port to use Python 3.9 is actually the right thing to do. 

Apologies @ferdy for the initial review - I was mistaken there.

Reply:
@cjones051073 duly noted and done! 
@reneeotten no worries! ;-)